tests/integration/test_textarea.py

```diff
@@ -27,3 +31,15 @@ def test_acceptable_submission(self):
 
         self.post(action=None)
         self.assertInUrl('thank-you')
+
+    def test_big_list_of_naughty_strings(self):
+        self.launchSurvey('test', 'big_list_naughty_strings')
+
+        answers = {}
+        for i in range(0, len(NAUGHTY_STRINGS)):
+            k = 'answer{}'.format(i)
+            answers[k] = NAUGHTY_STRINGS[i]
+
+        self.post(answers)
+        self.assertInUrl('summary')
+        self.assertEqual(200, self.last_response.status_code)
```

Comment:
Yes, this is a large grained test to make sure that the application doesn't fall over when handling "naughty" input. Finer grained tests, e.g. for XSS vectors, already exist. We should continue to review fine grained security test coverage as the platform evolves. For example, if we consider SQL injection to be a potential vector we should cover that in specific tests.